Editors/vscode/package-lock.json

```diff
@@ -1,882 +1,8 @@
 {
     "name": "sourcekit-lsp",
     "version": "0.0.1",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
I think if you update npm itself you will be able to use lockfile version 2 and won't see this change.  That's what I did anyway.

Reply:
Hmm, I installed the latest LTS node.js version.  Is that not the version to use?

Reply:
Updating nom outside of node.js seems to have done the trick.  I think that this deserves a note in the development notes, but that can live for another PR.